package/tvheadend-app-osmc/files/DEBIAN/postinst

```diff
@@ -41,6 +94,61 @@ if [ "$1" = "configure" ]; then
 		fi
 	fi
 
-	if ischroot; then exit 0; fi
+	# start tvh so that it updates any old user files
+	echo Starting tvheadend to finalise setup for osmc
 	if systemctl is-enabled tvheadend.service >/dev/null; then systemctl start tvheadend.service; fi
+	#systemctl restart tvheadend
+	sleep 5
```

Comment:
Thanks for giving it a go. I'm still not keen on starting TVH as part of the migration process. There's a lot that can go wrong here.

>Tried all day to make this work without starting tvh and it doesn't.

Okay -- so what does TVH exactly do when it starts that's doing the magic? We can pick up the migration steps from source and then work out what we need to do in Bash. I'd suggest taking a 4.0.9 config, starting TVH and diffing before and after. We can then find out what TVH is doing in terms of transformation.